lua/fzf.lua

```diff
@@ -32,14 +32,65 @@ local function get_lines_from_file(file)
   return t
 end
 
+-- can be overwritten by the user
```

Comment:
Is there a reason you didn't merge window_options and user_opts together? If user options are prefixed with `fzf_`, then there won't be a conflict and the code might be simpler?

Reply:
That was my original thought and preference too but since in the past you allowed users to set `default_window_options` (it’s in the [FAQ](https://github.com/vijaymarupudi/nvim-fzf#FAQ)), I figured I’d leave it be, any other approach that maintains backward compatibility is fine or you can scrap this part’s backward compatibility all together?